src/panel/favorites_panel.js

```diff
@@ -27,13 +33,41 @@ function Favorite() {
   PanelManager.register(this)
 }
 
+Favorite.prototype.toggleMore = function (position) {
+  if(this.openMoreMenuPosition === -1) {
```

Comment:
I think it should be allowed to open another menu (and close the existing one) by clicking on another favorite. Currently that requires two clicks, and that is a bit weird to me.